mozdownload/scraper.py

```diff
@@ -721,6 +751,28 @@ def __init__(self, branch='mozilla-central', build_number=None, date=None,
 
     def get_build_info(self):
         """Define additional build information."""
+        # Retrieve build by revision
```

Comment:
There's a lot in common between `TinderboxScraper.get_build_info` and `DailyScraper.get_build_info`. Want to refactor some of it into `Scraper`?